### ISSUE TITLE
PHP error with Neos 5.3

### ISSUE THREAD

**Issue by markusguenther** [Unikka/login-as#20]
### Description

The user @paavo submitted the issue via Slack

```
Return value of Unikka\LoginAs\Service\ImpersonateService_Original::getSessionData() must be an instance of Unikka\LoginAs\Service\mixed, null returned
```

**Comment by markusguenther** [comment]
This is a direct result of this change https://github.com/Unikka/login-as/commit/c785adcaeacf17fe03f91d99b2bacbb9e88a3476

**Comment by GuangHa** [comment]
We have the same problem, the problem occurs with Neos 7.3.5 and PHP 7.4.3.
If its with PHP 8.1.* it works. As a temporary solution we used the 2.0.2 version.

**Comment by paavo** [comment]
Any plans to "fix" this?

**Comment by paavo** [comment]
@markusguenther Can we please revert this change or move it to a new major version?
Every project where im  using `"unikka/login-as": "^2"` constraint fails after composer update 🙈

**Comment by markusguenther** [comment]
This has been released now with version 3.01. Thanks for all contributions 💙

**Comment by paavo** [comment]
With `"unikka/login-as": "^2"` dependency it still throws an error 500 after login. 
Wouldn't it make sense to push this back to version 2 @markusguenther?
